src/purchases.ts

```diff
@@ -1,22 +1,22 @@
-import { NativeEventEmitter, NativeModules } from "react-native";
```

Comment:
Happy to change .ts files to 4 spaces too, but I believe 2 spaces is more common in javascript world, correct?

Reply:
Not an expert but I think so yeah... It would also be ideal to have a linter for these, I think those help a ton. Not for this PR though.